oneshot.d/browser-cleanup-startup-cache

```diff
@@ -3,5 +3,5 @@
 rm -rf ~/.mozilla/mozembed/startupCache
 rm -rf ~/.mozilla/captiveportal/startupCache
 # New startup cache location
-rm -rf ~/.local/share/sailfish-browser/.mozilla/startupCache
```

Comment:
Are these paths actually correct? I would think that there needs to be `org.sailfishos` directory before `browser`.